ArduCopter/mode_guided.cpp

```diff
@@ -366,6 +366,11 @@ void Copter::ModeGuided::run()
 // guided_takeoff_run - takeoff in guided mode
 //      called by guided_run at 100hz or more
 void Copter::ModeGuided::takeoff_run()
+{
+    auto_takeoff_run();
+}
+
+void Copter::Mode::auto_takeoff_run()
```

Comment:
shouldn't this be move in takeoff.cpp ?

Reply:
We should probably split into `user_takeoff.cpp` and `auto_takeoff.cpp`, I think.  But not as part of this PR :-)